### PR TITLE
Create Calendar for Generic Zmanim

### PR DESCRIPTION
**Pull request by froco93**

### CODE REVIEW

File: ZmanimCalendarCalculator/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
```
Comment (froco93) [inline ZmanimCalendarCalculator/Properties/AssemblyInfo.cs]: probably need to change this

File: ZmanimCalendarCalculator/ZmanimCalendarCreator.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            return zmanimResultList;
+        }
+
+        private bool ShouldAddZmanToCalandar(EngineResultDay zmanimforDay)
+        {
+            if (zmanimforDay.Time.IsShabbos)
+            {
+                return true;
+            }
+            if(zmanimforDay.Time.IsYomTov && !zmanimforDay.Time.TomorrowNightIsYomTov)
```
Comment (eschramm3) [inline ZmanimCalendarCalculator/ZmanimCalendarCreator.cs]: if you're just interested in havdallah times, we wouldn't care about yontifs that are immediately followed by shabbos

File: ZmanimCalendarCalculator/ZmanResult.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp3
+{
+    public class ZmanResult
+    {
+        public DateTime EndTime { get; set; }
```
Comment (eschramm3) [inline ZmanimCalendarCalculator/ZmanResult.cs]: I would call this something else like "HavdallahTime" but not so important

File: ZmanimCalendarCalculator/ZmanResult.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+        public string ParshaAndHoliday { get; set; }
```
Comment (eschramm3) [inline ZmanimCalendarCalculator/ZmanResult.cs]: what values does this take?

File: ZmanimCalendarCalculator/ZmanResult.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+        public bool IsShabbat { get; set; }
```
Comment (eschramm3) [inline ZmanimCalendarCalculator/ZmanResult.cs]: IsYomTov as well?

**Pull request merged**